File: Python/product.py
class product:
    def __init__(self, idProduct="-", brand="-", nama='-', harga='-',):
        # Constructor method to initialize idProduct, brand, nama, and harga attributes
        self.__idProduct = idProduct
        self.__brand = brand
        self.__nama = nama
        self.__harga = harga
    
    def setbrand(self, brand):
        # Method to set the brand attribute
        self.__brand = brand

    def getbrand(self):
        # Method to get the value of the brand attribute
        return self.__brand

    def setidProduct(self, idProduct):
        # Method to set the idProduct attribute
        self.__idProduct = idProduct

    def getidProduct(self):
        # Method to get the value of the idProduct attribute
        return self.__idProduct
	
    def setnama(self, nama):
        # Method to set the nama attribute
        self.__nama = nama
	
    def getnama(self):
        # Method to get the value of the nama attribute
        return self.__nama
    
    def setharga(self, harga):
        # Method to set the harga attribute
        self.__harga = harga
	
    def getharga(self):
        # Method to get the value of the harga attribute
        return self.__harga

File: PHP/product.php
<?php

// Definisi kelas Product
class product
{
	// Properti privat untuk idProduct, nama, brand, dan harga
	private $idProduct;
	private $nama;
	private $brand;
    private $harga;
	
	// Konstruktor kelas Product
	public function __construct(){
		// Menginisialisasi properti dengan nilai default
		$this->idProduct = "-";
		$this->nama = "-";
		$this->brand = "-";
        $this->harga = "-";
	}
	
	// Metode untuk mengatur nilai idProduct
	public function setidProduct($idProduct){
		$this->idProduct = $idProduct;
	}
	
	// Metode untuk mendapatkan nilai idProduct
	public function getidProduct(){
		return $this->idProduct;
	}
	
	// Metode untuk mengatur nilai nama
	public function setnama($nama){
		$this->nama = $nama;
	}
	
	// Metode untuk mendapatkan nilai nama
	public function getnama(){
		return $this->nama;
	}
	
	// Metode untuk mengatur nilai brand
	public function setbrand($brand){
		$this->brand = $brand;
	}
	
	// Metode untuk mendapatkan nilai brand
	public function getbrand(){
		return $this->brand;
	}
	
	// Metode untuk mengatur nilai harga
    public function setharga($harga){
		$this->harga = $harga;
	}
	
	// Metode untuk mendapatkan nilai harga
	public function getharga(){
		return $this->harga;
	}
}

?>
